ros2param/test/test_api.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rcl_interfaces.msg import ParameterType
+from ros2param.api import get_parameter_value
+
+
+def test_bool():
```

Comment:
All these tests are quite similar, so we could merge them in a single parametrized test, that is easier to maintain. Something like 

```python 

import pytest

@pytest.mark.parametrize("string_value,value_attribute,expected_type,expected_value", [
    ('true', 'bool_value', ParameterType.PARAMETER_BOOL, True),
    ('foo', 'string_value', ParameterType.PARAMETER_STRING, 'foo'),
   ('["foo", "bar", "buzz"]', 'string_array_value', ParameterType.PARAMETER_STRING_ARRAY, ['foo', 'bar', 'buzz'])
])
def test_get_parameter_value(string_value, value_attribute, expected_type, expected_value):
    value = get_parameter_value(string_value=string_value)
    assert value.type == expected_type
    assert getattr(value, value_attribute) is expected_value
```

Reply:
I like that, doing that now.